Runtime/Scripts/BaseMeshSync.cs

```diff
@@ -937,13 +949,10 @@ private void UpdateTextureAsset(TextureData src)
             // write data to file and import
             string path = m_assetsFolder + "/" + src.name;
             
-            if (!GetConfigV().GetModelImporterSettings().ImportTextures && File.Exists(path))
-            {
-                return;
-            }
+            bool needImport = GetConfigV().GetModelImporterSettings().ImportTextures || !File.Exists(path);
             
-            if (src.WriteToFile(path))
-                doImport(path);
+            if (!needImport || src.WriteToFile(path))
+                doImport(path, needImport);
```

Comment:
I am wondering why we need to add `!needImport` check.
Can we call it like this ?

```
if (src.WriteToFile(path))
    doImport(path, needImport);
```

Reply:
I was trying to avoid writing the file if it's not needed but it doesn't seem to slow it down so I've changed it now.